API_README.md

```diff
@@ -162,6 +163,7 @@ This is a listing of scopes:
 - `api:fhir` (user fhir which are the /fhir/ endpoints)
   - `user/AllergyIntolerance.read`
   - `user/CareTeam.read`
+  - `user/Coverage.read`
```

Comment:
i'd usually let this go, but since there's another minor change to do, might as well alphabetize (place below Condition) this as well :)

Reply:
Hoping that my next PR will have less of these kind of mistakes :)